pkg/controller/eventing/inmemory/channel/reconcile_test.go

```diff
@@ -542,7 +570,11 @@ func TestReconcile(t *testing.T) {
 	}
 }
 
-func makeChannel() *eventingv1alpha1.Channel {
+func makeChannel(pn ...string) *eventingv1alpha1.Channel {
+	provisionerName := ccpName
```

Comment:
Clever way to default. Move it to a separate function and add a comment saying what you are doing.

Reply:
done